backend/src/main/java/de/teamprojectfinance/backend/TransactionController.java

```diff
@@ -28,15 +31,25 @@ public Transaction addTransaction(@Valid @RequestBody TransactionWithNoId transa
     }
 
     @PutMapping("{id}")
-    public Transaction updateTransaction(@PathVariable("id") String id, @Valid @RequestBody TransactionWithNoId transactionWithNoId){
+    public Transaction updateTransaction(@PathVariable("id") String id, @Valid @RequestBody TransactionWithNoId transactionWithNoId) {
 
         return transactionService.updateTransaction(transactionWithNoId, id);
 
     }
 
     @DeleteMapping("{id}")
-    public void deleteTransaction(@PathVariable("id") String id){
+    public void deleteTransaction(@PathVariable("id") String id) {
         transactionService.deleteTransaction(id);
     }
 
-}
+
+    @ExceptionHandler({NoSuchElementException.class})
```

Comment:
```suggestion
    @ExceptionHandler(NoSuchElementException.class)
```

Bei nur einer Exception sind die Klammern nicht nötig